pandas/tests/frame/test_combine_concat.py

```diff
@@ -167,6 +167,11 @@ def test_append_list_of_series_dicts(self):
         expected = df.append(DataFrame(dicts), ignore_index=True, sort=True)
         assert_frame_equal(result, expected)
 
+        dicts = [{'foo': 9}, {'bar': 10}]
+        result = df.append(dicts, ignore_index=True, sort=True)
+        expected = df.append(DataFrame(dicts), ignore_index=True, sort=True)
+        assert_frame_equal(result, expected)
+
```

Comment:
Separate out test and reference issue number as a comment under test function `def`.